### PR TITLE
Polkadot events

### PR DESCRIPTION
**Pull request by jakai7**
Adding methods for listening to events emitted by the Parachain.

### CODE REVIEW

File: src/net/polkadot.ts [hunk 1]
```diff
@@ -132,6 +134,48 @@ export default class Polkadot extends Api {
...
+              // PolkaEth amount
+              amount: event.data[2].toString(),
+            });
+
+            // Checks if the parachain emited event is for a burned
+          } else if (event.section === 'asset' && event.method === 'Burned') {
+            console.log('Got Assets.Minted event');
+
+            // Notify local transaction object the asset is burned
+            this.net.polkaEthBurned({
```
Comment (musnit) [inline src/net/polkadot.ts:164]: where is this function?

File: src/net/polkadot.ts [hunk 2]
```diff
@@ -132,6 +134,48 @@ export default class Polkadot extends Api {
...
+        // Loop through the events in the current block
+        events.forEach((record) => {
+          const { event } = record;
+
+          // Checks if the parachain emited event is for a Minted asset
+          if (event.section === 'asset' && event.method === 'Minted') {
+            console.log('Got Assets.Minted event');
+
+            // Notify local transaction object the asset is minted
+            this.net.polkaEthMinted({
```
Comment (musnit) [inline src/net/polkadot.ts:152]: where is this function?

File: src/net/polkadot.ts [hunk 3]
```diff
@@ -132,6 +134,48 @@ export default class Polkadot extends Api {
...
+              // Receiver of the sent PolkaEth
+              AccountId: event.data[1].toString(),
+              // PolkaEth amount
+              amount: event.data[2].toString(),
+            });
+
+            // Checks if the parachain emited event is for a burned
+          } else if (event.section === 'asset' && event.method === 'Burned') {
+            console.log('Got Assets.Minted event');
```
Comment (musnit) [inline src/net/polkadot.ts:161]: should be Assets.Burned

File: src/net/index.ts [hunk 4]
```diff
@@ -1,9 +1,12 @@
+import web3 from 'web3';
+
 import Eth from './eth';
 import Polkadot from './polkadot';
 
 import { Dispatch } from 'redux';
 
 import { REQUIRED_ETH_CONFIRMATIONS } from '../config';
+import BigNumber from 'bignumber.js';
```
Comment (musnit) [inline src/net/index.ts:9]: can remove this now

Comment (jakai7) [reply src/net/index.ts]: Removed it.

**Pull request merged**